db/migrate/20170607200956_remove_null_value_constrain_from_schools.rb

```diff
@@ -1,5 +0,0 @@
-class RemoveNullValueConstrainFromSchools < ActiveRecord::Migration[5.0]
-  def change
-    change_column :schools, :survey_active, :boolean, :null => true
```

Comment:
Removal here may cause issues with the walkshed generation, only if the constraint persists on the table itself or if in migrating the data, it expects something.